constellations/gemini-admin/src/home/home.js

```diff
@@ -1,205 +1,110 @@
-                    : "",
-            rows: semesters,
-            selection: "semesters",
-            switch: semesters.length != 0,
-        });
-    }
-
-    onClickSessions() {
-        let sessions = this.state.sessions.map((row, index) => {
-            return <DashboardRow key={index} title={row.sessionId} />;
-        });
-        this.setState({
-            noUnpub:
-                sessions.length == 0
-                    ? "All items are published! Your website is up to date!"
-                    : "",
-            rows: sessions,
-            selection: "sessions",
-            switch: sessions.length != 0,
+    render() {
+        let unpublishedClasses = this.state.classes.map((row, index) => {
+            return <li key={index}> {row.classId} </li>;
         });
-    }
 
-    onClickPage() {
-        window.location.hash = this.state.selection;
-        console.log("Go to Page clicked");
-    }
-
-    onClickPublishAll() {
-        console.log("Publish All!");
-    }
-
-    render() {
-        let unpubMessage = <div> </div>;
-        if (this.state.switch) {
-            unpubMessage = (
-                <h4>
-                    {" "}
-                    You have {this.state.rows.length} unpublished{" "}
-                    {this.state.selection}{" "}
-                </h4>
-            );
-        }
         return (
             <div id="view-home">
-                <div className="home-left">
-                    <h2> Unpublished Content </h2>
-                    <ul>
-                        <button onClick={this.onClickPrograms}>Programs</button>
-                        <button onClick={this.onClickClasses}>Classes</button>
-                        <button onClick={this.onClickLocations}>
-                            Locations
-                        </button>
-                        <button onClick={this.onClickAchievements}>
-                            Achievements
-                        </button>
-                        <button onClick={this.onClickSemesters}>
-                            Semesters
+                <h1>Administrator Dashboard</h1>
+
+                <div class="section">
+                    <div class="container-class">
+                        <span>
+                            <h3>Unpublished Classes</h3>{" "}
+                        </span>
+                        <button id="publish">
+                            <Link to={"/classes"}>View Details to Publish</Link>
                         </button>
-                        <button onClick={this.onClickSessions}>Sessions</button>
-                    </ul>
-                    <h2> Registrations </h2>
-                    <ul>
-                        <button>New Users</button>
-                        <button>Questions</button>
-                        <button>Complaints</button>
-                    </ul>
+                    </div>
+                    <div class="class-section">
+                        <p>Class ID</p>
```

Comment:
Just replace this with a `<div>` and give it a className (i.e. `<div className="list-header-item"`) to add the bottom underline under it. `<div>` are block elements which mean they take up the whole row, which is why `<p>` elements work. `<span>` elements are inline elements which means they can be placed adjacent to other inline elements.